regparser/notice/xml.py

```diff
@@ -142,7 +142,14 @@ def set_agencies(self, agencies=None):
         # that contain subagencies in children fields:
         for agency in agencies:
             agency["children"] = []
-        agency_map = {agency["id"]: agency for agency in agencies}
+#        agency_map = {agency["id"]: agency for agency in agencies}
+        agency_map = {}
+        for agency in agencies:
+            try:
+                agency_map[agency["id"]] = agency
```

Comment:
Happens because of malformed agency entries, is it better to error loudly for fixing?